Cake.Issues.Recipe/Content/build.cake

```diff
@@ -113,16 +115,31 @@ IssuesBuildTasks.CreateFullIssuesReportTask = Task("Create-FullIssuesReport")
         IssuesParameters.OutputDirectory.CombineWithFilePath(reportFileName);
     EnsureDirectoryExists(IssuesParameters.OutputDirectory);
 
+    IIssueReportFormat issueFormat = null;
+
+    if (!Context.Environment.Runtime.IsCoreClr)
```

Comment:
Loading of addins should mainly be triggered by parameters. Meaning that if any parameter is set which requires the reporting addin it should be loaded. In case of the reporting addin which is currently not compatible with .NET Core we can decide what to do in this case (IMHO it should be an error, since the user says he want's report and the script is not capable of creating one).

Reply:
Well, I didn't want to implement any changes to the options class for this PR, which is why I just added a check if we are running on .NET Core or not.

But I do agree that it should be triggered by parameters.